eng/devices/catalyst.cake

```diff
@@ -65,10 +67,12 @@ Task("uitest")
 
 RunTarget(TARGET);
 
-void ExecuteBuild(string project, string binDir, string config, string rid, string tfm, string toolPath)
+void ExecuteBuild(string project, string binDir, string config, string rid, string tfm, string toolPath, bool useCoreClr)
 {
 	var projectName = System.IO.Path.GetFileNameWithoutExtension(project);
-	var binlog = $"{binDir}/{projectName}-{config}-catalyst.binlog";
+	bool isUsingCoreClr = useCoreClr.ToString().Equals("true", StringComparison.CurrentCultureIgnoreCase);
+	var monoRuntime = isUsingCoreClr ? "coreclr" : "mono";
```

Comment:
You're converting a bool to a string to compare it to "true"? Can't it just be simplified to:

```suggestion
	var monoRuntime = useCoreClr ? "coreclr" : "mono";
```